packages/vaadin-text-field/test/validation.test.js

```diff
@@ -163,8 +162,6 @@ import '../vaadin-text-field.js';
 const fixtures = [
   { name: 'number field', tpl: '<vaadin-number-field invalid></vaadin-number-field>' },
   { name: 'number field with value', tpl: '<vaadin-number-field invalid value="42"></vaadin-number-field>' },
-  { name: 'password field', tpl: '<vaadin-password-field invalid></vaadin-password-field>' },
-  { name: 'password field with value', tpl: '<vaadin-password-field invalid value="foo"></vaadin-password-field>' },
```

Comment:
Didn't find these test cases in the new package. They most likely need to be moved as well as it was done for the email field.

Reply:
Here is a similar discussion for the email field: https://github.com/vaadin/web-components/pull/2320#discussion_r692184831

Reply:
Added the tests.